### ISSUE TITLE
confusing explanation

### ISSUE THREAD

**Issue by Bjarten** [udacity/DL_PyTorch#3]
https://github.com/udacity/DL_PyTorch/blob/ff9bf7ad7b909621b52c570e3505618af8407e8c/Part%203%20-%20Training%20Neural%20Networks.ipynb#L232

"Often, the output will have values really close to zero or really close to one. Due to inaccuracies with representing numbers as floating points, computations with a softmax output can lose accuracy and become unstable. To get around this, we'll use the raw output, called the logits, to calculate the loss."

The description for the nn.CrossEntropyLoss() say: "This criterion combines :func:`nn.LogSoftmax` and :func:`nn.NLLLoss` in one single class.". 

I'm a bit confused about this explanation, since the nn.CrossEntropyLoss() criterion contains a nn.LogSoftmax function.

**Comment by WikiGenius** [comment]
Hello @Bjarten 
I am confused too, did you discovered the answer?
